code/game/machinery/Sleeper.dm

```diff
@@ -245,10 +248,9 @@
 			if(occupant)
 				to_chat(user, "<span class='warning'>\The [src] is already occupied.</span>")
 				return TRUE
-			var/bucklestatus = L.bucklecheck(user)
-
-			if (!bucklestatus)//incase the patient got buckled_to during the delay
-				return TRUE
+			if(bucklestatus == 2)
+				var/obj/structure/LB = L.buckled_to
+				LB.user_unbuckle(user)
```

Comment:
What's happening here? We're unbuckling the victim if the grabber is a robot - which can't happen because robots can't grab? Am I misunderstanding?

Side-note : `/mob/living/proc/bucklecheck(var/mob/living/user)` is a weird proc and magic numbers suck

Reply:
Correct, for both statements! The ``proc/bucklecheck`` is goddamn weird, and that the unbuckling happens if the robots somehow had them in a grab. I put it in because I got paranoid about it ever happening since I was already having some trouble testing the stationbounds

I'll remove those from the grab check for the various machines

Reply:
Done!